imports/plugins/core/cart/server/no-meteor/mutations/convertAnonymousCartToNewAccountCart.js

```diff
@@ -39,7 +41,8 @@ export default async function convertAnonymousCartToNewAccountCart({
   };
 
   if (anonymousCart.referenceId) {
-    newCart.referenceId = anonymousCart.referenceId;
+    await Cart.findOneAndUpdate({ _id }, {$unset: { referenceId: 1 } });
```

Comment:
Can you add a comment above this line about why this is being done? i.e., because referenceId has a uniqueness constraint but we want to copy the same value from anonymous cart to account cart.